client/src/stories/mainLogo.js

```diff
@@ -0,0 +1,22 @@
+import styled from "styled-components/macro";
+
+export const MainLogo = styled.img`
+  width: 175px;
+  display: grid;
+  grid-area: image;
+`;
+
+export const Header = styled.h1`
```

Comment:
If you use text as part of a logo, I would not put ist as an h1. An h1 is an element, that only occurs once one the page. This is also a semantic aspect for search engines like google. Rather than that, I would set it as a span element or a div. That enables more possibilities for styling, since the h1 always comes with a sort of default styling.  

You create a styled component out of each element, that belongs to the logo. Instead, put it all together and group it into one component. If you have questions about it, you can ask me.

Reply:
Thanks for the advice!